framework/src/base/Moose.C

```diff
@@ -1193,3 +1193,5 @@ bool _deprecated_is_error = false;
 bool _throw_on_error = false;
 
 } // namespace Moose
+
+OStreamProxy & _console = Moose::out;
```

Comment:
I should not have used the word static, this is a *global* variable.  The [rule](http://stackoverflow.com/questions/3746238/c-global-initialization-order-ignores-dependencies/3746249#3746249) appears to be that global variables in a single translation unit are initialized in the order in which they are defined.  Assuming that `std::cout` is also a global variable of some kind, this should be fine.  Still not a big fan of the shadowing, however.